src/audio/host-zephyr.c

```diff
@@ -950,10 +950,20 @@ static int host_params(struct comp_dev *dev,
 	return err;
 }
 
+int host_zephyr_prepare(struct host_data *hd)
+{
+	struct comp_buffer __sparse_cache *buf_c;
+
+	buf_c = buffer_acquire(hd->dma_buffer);
```

Comment:
can you move this to the same line as the declaration on 955

Reply:
ok